components/StoryCard.tsx

```diff
@@ -28,6 +31,13 @@ const StyledSpan = styled.span`
   color: #3291ff;
 `;
 
+const Title = styled.textarea`
```

Comment:
@shc023 @RobertoValle31  @jansen0825 - do you know a way I can have the textarea resize as the content gets bigger? it seems to be fixed

Reply:
native Textarea you'll have to use JS for that. Geist does have a `TextArea` component but it looks like that also doesn't auto-expand..

Reply:
@shc023 makes sense. Did you try forcing it to a 100% width?

Reply:
Ok, I'll leave that to you @shc023 in a future pr, will merge for now

Reply:
@mattvv  Maybe you can do this. Use a div and add the 'role' prop. You will need to add some border to the div in order to make it visible. You can find more info [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/textbox_role)
<img width="1170" alt="Screen Shot 2020-10-15 at 2 02 32 PM" src="https://user-images.githubusercontent.com/22966249/96168933-73573600-0eef-11eb-9c83-2ce7143c8584.png">